crates/wasm_demo/README.md

```diff
@@ -0,0 +1,12 @@
+# Rust Analyzer WASM demo
+
+## building
+
+```sh
+cd crates/wasm_demo
+wasm-pack build .
+
+cd www
+npm i
+npx webpack-dev-server --host 0.0.0.0
```

Comment:
Ideally, we should build a static page and host it on GH pages, like here: https://github.com/matklad/tom/tree/gh-pages